core/lib/types/src/api/mod.rs

```diff
@@ -642,15 +642,28 @@ impl From<Call> for DebugCall {
 
 #[derive(Default, Serialize, Deserialize, Clone, Debug)]
 pub struct ProtocolVersion {
```

Comment:
Probably it makes sense to add a comment explaining that this structure is in "migration" phase, and what should be changed and when.
Otherwise we risk finding ourselves asking "Why was it done this way?" X month later (not for the first time).